test/e2e/multihoming.go

```diff
@@ -636,6 +642,48 @@ var _ = Describe("Multi Homing", func() {
 			),
 		)
 
+		Context("localnet OVN-K secondary network", func() {
+			const (
+				clientPodName     = "client-pod"
+				nodeHostnameKey   = "kubernetes.io/hostname"
+				servicePort       = 9000
+				dockerNetworkName = "underlay"
+			)
+
+			table.DescribeTable(
+				"with a service running on the underlay",
+				func(netConfigParams networkAttachmentConfigParams, clientPodConfig podConfiguration) {
+					netConfig := newNetworkAttachmentConfig(netConfigParams)
+					netConfig.topology = "localnet"
+					netConfig.cidr = secondaryLocalnetNetworkCIDR
+					netConfig.excludeCIDRs = []string{secondaryLocalnetExcludeCIDR}
```

Comment:
why don't you just pass this on the `netConfigParams` variable ?

Reply:
since it is not a free variable, didnt want it as part of the parameters
anyhow if we ditch the Table it will be outdated